build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
why do we need a separate build script for the logger? why not use the existing Cmake?